src/app-bundles/radar-time-series-bundle.js

```diff
@@ -1,7 +1,6 @@
-import createRestBundle from "./create-rest-bundle";
-import { getRestUrl } from "./bundle-utils";
 import { createSelector } from "redux-bundler";
-import { LOCATIONDETAIL_SET_CODE } from "./location-detail-bundle";
+import createRestBundle from "./create-rest-bundle";
+import { getRestUrl, getIntervalTime } from "./bundle-utils";
```

Comment:
Hmm on my branch when I pulled down, I don't see the getIntervalTime func in bundle-utils. Did it not get pushed up?

Reply:
getIntervalTime isn't part of the bundle-utils, it comes with the createNestedUrlBundle which is part of cormaps. It confused me at first too

Reply:
ahh yea sorry, forgot to push it up

Reply:
It should be pushed up now